data/exploits/psnuffle/url.rb

```diff
@@ -1,22 +1,24 @@
-# Psnuffle password sniffer add-on class for HTTP GET URL's
+# Psnuffle password sniffer add-on class for HTTP URLs
 # part of psnuffle sniffer auxiliary module
+
 #
-# Very simple example how to write sniffer extensions
+# Sniffer class for GET/POST URLs.
+# Also extracts HTTP Basic authentication credentials.
 #
-
-# Sniffer class for GET URL's
 class SnifferURL < BaseProtocolParser
   def register_sigs
     self.sigs = {
-      :get		=> /^GET\s+([^\n]+)\s+HTTP\/\d\.\d/i,
-      :webhost	=> /^HOST\:\s+([^\n\r]+)/i,
+      :get        => /^GET\s+([^\n]+)\s+HTTP\/\d\.\d/i,
```

Comment:
Prefer `get:`, etc.

Reply:
Assuming you're keeping the style because no one really uses this anymore anyway.